yt_dlp/extractor/arte.py

```diff
@@ -217,43 +216,39 @@ class ArteTVPlaylistIE(ArteTVBaseIE):
     _VALID_URL = r'https?://(?:www\.)?arte\.tv/(?P<lang>%s)/videos/(?P<id>RC-\d{6})' % ArteTVBaseIE._ARTE_LANGUAGES
     _TESTS = [{
         'url': 'https://www.arte.tv/en/videos/RC-016954/earn-a-living/',
-        'info_dict': {
-            'id': 'RC-016954',
-            'title': 'Earn a Living',
-            'description': 'md5:d322c55011514b3a7241f7fb80d494c2',
-        },
-        'playlist_mincount': 6,
+        'only_matching': True,
     }, {
         'url': 'https://www.arte.tv/pl/videos/RC-014123/arte-reportage/',
+        # too long to comfortably test
```

Comment:
?

Reply:
100 videos.

Reply:
playlist tests don't need to test the underlying videos - just the playlist's count and metadata. Eg: https://github.com/yt-dlp/yt-dlp/blob/f82711587cee043cb2496fe180b5cc0e07c06eda/yt_dlp/extractor/youtube.py#L4504-L4521

If you want to be more rigorous, you can set `playlist_mincount` to a low number and test only the first few entries. Eg: https://github.com/yt-dlp/yt-dlp/blob/86e5f3ed2e6e71eb81ea4c9e26288f16119ffd0c/yt_dlp/extractor/chingari.py#L103-L176

You could also add `'params': {'flat_playlist': True}` to futher limit the scope of the test

Reply:
I know, but running the test will extract all videos regardless. Try it yourself:

```sh-session
$ python3 test/test_download.py TestDownload.test_ArtePlaylistTV
```

Reply:
Okay, I was wrong. I don’t know why it did that before.